test/jdk/security/infra/java/security/cert/CertPathValidator/certification/QuoVadisCA.java

```diff
@@ -57,10 +58,12 @@ public static void main(String[] args) throws Exception {
 
 class RootCA1G3 {
 
-    // Owner: CN=DigiCert QuoVadis TLS ICA QV Root CA 1 G3, O="DigiCert, Inc", C=US
+    // Owner: CN=DigiCert QuoVadis TLS ICA QV Root CA 1 G3, O="DigiCert,
+    // Inc", C=US
```

Comment:
Can you put the "Inc", C=US" at the end of the previous line instead of on a new line, or alternatively indent it so it is under "CN:"? I think it would be more readable. Same comment applies to all the test certificates.